data/windows.yaml

```diff
@@ -3347,3 +3360,104 @@ sources:
 labels: [System, Network]
 supported_os: [Windows]
 urls: ['https://technet.microsoft.com/en-us/library/dd197418(v=ws.10).aspx']
+---
+name: WindowsCryptnetUrlCacheMetadata
+doc: |
+  Metadata of a Windows cache of files downloaded from the internet.
+
+  Helpful when investigating the use of "Living of the Land" tools that allow
+  attackers to download arbitrary files from the internet, such as
+  "certutil.exe".
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_systemroot%%\System32\config\systemprofile\AppData\LocalLow\Microsoft\CryptnetUrlCache\MetaData\*'
+    - '%%environ_systemroot%%\SysWOW64\config\systemprofile\AppData\LocalLow\Microsoft\CryptnetUrlCache\MetaData\*'
+    - '%%users.localappdata_low%%\Microsoft\CryptnetUrlCache\MetaData\*'
+    separator: '\'
+labels: [System, Network]
+conditions: [os_major_version >= 6]
+supported_os: [Windows]
+urls:
+- 'https://u0041.co/blog/post/3'
```

Comment:
Seeing the duplication of URLs, why not create a forensicswiki article about cryptneturlcache and link there? Also links of blogs and source typically changes over time, so I would recommend to link to one article that describes this artifact.

Reply:
The URLs are repeated because the entire artifact is a copy-and-paste, with the only change being the last folder (`Content` vs `MetaData`). The distinction is important though as the Content folder can be quite large in size, while MetaData will always only retain metadata, hence its small size.

I'll update the ForensicWiki page about Windows artifacts [1] once the account registration feature is working again [2]. Could we merge the changes without forensicswiki and I'll come back to it and update it once the issue with ForensicWiki is solved?

1.  https://forensicswiki.xyz/wiki/index.php?title=Windows#Forensics
2.  https://groups.google.com/g/forensicswiki-reborn/

Reply:
The URL [2] above suggests that the values `%%users.*%%` are populated by lookups in 3 registry keys. I just checked and non of them (or the entire user registry hive for that matter) contain an entry that would populate `localappdata_low`, so I'll revert it back to a direct path.